src/runner.js

```diff
@@ -151,26 +151,32 @@ class MoleculerRunner {
 	loadConfigFile() {
 		let filePath;
 		// Env vars have priority over the flags
-		if (process.env["MOLECULER_CONFIG"]) {
-			filePath = path.isAbsolute(process.env["MOLECULER_CONFIG"])
-				? process.env["MOLECULER_CONFIG"]
-				: path.resolve(process.cwd(), process.env["MOLECULER_CONFIG"]);
-		} else if (this.flags.config) {
-			filePath = path.isAbsolute(this.flags.config)
-				? this.flags.config
-				: path.resolve(process.cwd(), this.flags.config);
+		const configPath = process.env["MOLECULER_CONFIG"] || this.flags.config;
+
+		if (configPath != null) {
```

Comment:
Please, use the strict equality operator (`===`)

Reply:
I can make this change if that's the standard, but generally speaking my one exception to strict vs. loose equality checking is against `null` as loose equality will capture both `undefined` and `null` scenarios.  All other scenarios I would generally use strict equality instead, but this is done intentionally.  It looks like that behavior is used often in this repo (434 results for `!= null` and 63 results for `== null`).

There's actually an eslint mode for their strict equality rule to allow for this (https://eslint.org/docs/latest/rules/eqeqeq#smart) although it doesn't look like the "eqeqeq" rule is enabled for this repo.

As I said, I can make the change if that's the preference here.

Reply:
Thank you! If there can be `undefined` or `null`, then it's better to check it explicitly in the code.

PS: I would add a rule in eslint and check the entire repository for this in a separate PR. I consider this an important step to improve the quality of the code.

@icebob, what do you think?

Reply:
In fact, in a large amount of code with many such checks, debugging unexpected behaviour at the end can be an unpleasant surprise at one of these checks.

Reply:
@intech I think it's a very good idea to turn on the `eqeqeq` eslint rule in the repo.  If the standard for the repo is to use strict equality its much better to catch that in lint than in PR review.

With that said, I'm personally a fan of allowing for the loose equality exception against `null`. In the vast majority of circumstances, `null` and `undefined` are treated equivalently and the only things `null` matches against loosely are itself and `undefined`.  I feel like its much more likely to introduce a bug for a value that could be either `null` or `undefined` and forgetting to check against both when using strict equality than it is to have a scenario where `null` and `undefined` are actually treated differently from each other and a bug is introduced that way.

Ultimately I'm good with either approach here, but my opinion is that loose equality against `null` is a good exception to the strict equality rule.

Reply:
I don't like that forcing the conditions to `===`. As @shawnmcknight said, many places allow that `null` and `undefined` handled by the same way. When it's not true, I use the `===` instead of `==` in the codebases. And I check these situations in the PRs as well.

For me the `a == null` it's more readable than the `a === null || a === undefined`.